File: 4000~4999/4344.py
C = int(input())
for i in range(C):
    N = list(map(int, input().split()))

    cnt = 0
    avg = sum(N[1:]) / N[0]
    for j in N[1:]:
        if j > avg:
            cnt += 1

    print("{0:.3f}%".format(cnt / N[0] * 100))

File: c/4000~4999/4344.c
#include <stdio.h>

int main()
{
	int c = 0;
	scanf("%d", &c);

	for (int i = 0; i < c; i++)
	{
		int n = 0;
		int sum = 0;
		int scores[1000] = {0};

		scanf("%d", &n);
		for (int j = 0; j < n; j++)
		{
			scanf("%d", &scores[j]);
			sum += scores[j];
		}

		int cnt = 0;
		double avg = (double)sum / n;
		for (int j = 0; j < n; j++)
		{
			if (scores[j] > avg) cnt++;
		}
		printf("%.3f%%\n", (double)cnt / n * 100);
	}
	return 0;
}
